upf/upf.c

```diff
@@ -195,6 +195,24 @@ vnet_upf_nat_pool_add_del (u8 * nwi_name, ip4_address_t start_addr,
   return 0;
 }
 
+void
+upf_pfcp_policer_init (upf_main_t * vm)
```

Comment:
`upf_main_t *` vars are (historically) called `gtm` or sometimes `sm`, but never `vm` which is used for `vlib_main_t *`. Perhaps someday we should rename them to `um`, but at least let's not make the naming even more diverse

Reply:
this thing has been refactored. Probably another review of the PR is required.